api/response/schema_query.py

```diff
@@ -112,10 +113,48 @@ def filter_rooms_by_responses(self, info, upper_limit, lower_limit):
                 filtered_responses.append(response)
         return filtered_responses
 
+    def search_response_by_room(
```

Comment:
The @caveinn  who worked on the filter functionality introduced filter_by as one of the arguments, is there any place where this is used to do the actual filter? If not then we don't need it.

Reply:
@Bonifase, Ok, I will work on this

Reply:
@Bonifase , I have implemented this feedback